keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -77,7 +91,26 @@ data class ServerGroup(
   override val disabled: Boolean,
   override val instanceCounts: InstanceCounts,
   override val createdTime: Long
-) : BaseEc2ServerGroup
+) : BaseEc2ServerGroup {
+  init {
+    ensureLaunchConfigInfoIsPresent(this)
+  }
+}
+
+/**
+ * Clouddriver should always return either a launchConfig field or a launchTemplate
+ * field for an EC2 server group.
+ *
+ * This is a helper function that asserts that one of these fields must be present.
+ * It is intended to be called from the init block of classes that implement BaseEc2ServerGroup
+ */
+fun ensureLaunchConfigInfoIsPresent(sg : BaseEc2ServerGroup) {
+  sg.run {
```

Comment:
why do we need here `sb.run`?

Reply:
To be honest, I don't remember why I put it there to begin with, but we don't need it. I removed it.